lib/ingram_micro/base_element.rb

```diff
@@ -1,18 +1,22 @@
 class IngramMicro::BaseElement
   attr_accessor :element
 
-  def initialize(options={})
+  def initialize(options = {})
     @element = {}
-    self.defaults.each do |field, default|
+    defaults.each do |field, default|
       value = (options[field] ? options[field] : default)
       @element[field] = value
     end
-    @element[:customer_id] ||= IngramMicro.configuration.customer_id
```

Comment:
wouldn't removing this be problematic? are you sure this doesn't break the xml we send to IM